core/common/src/main/java/alluxio/uri/StandardURI.java

```diff
@@ -257,12 +217,7 @@ public int hashCode() {
     } else {
       hashCode = URIUtils.hash(hashCode, mPath);
       hashCode = URIUtils.hash(hashCode, mQuery);
-      if (mHost != null) {
-        hashCode = URIUtils.hashIgnoreCase(hashCode, mHost);
-        hashCode += 1949 * mPort;
-      } else {
-        hashCode = URIUtils.hash(hashCode, mAuthority);
-      }
+      hashCode = URIUtils.hash(hashCode, mAuthority.toString());
```

Comment:
use the hash of the authority

Reply:
here I could not use the ```mAuthority.hashCode()```, since the second parameter could only be String.